src/org/opendatakit/briefcase/ui/ExportPanel.java

```diff
@@ -358,6 +389,16 @@ public ExportPanel(TerminationFuture terminationFuture) {
     btnPemFileChooseButton = new JButton("Choose...");
     btnPemFileChooseButton.addActionListener(new PEMFileActionListener());
 
+    JLabel lblDateRange = new JLabel("Specific Date Range:");
+    checkDates = new JCheckBox("", false);
```

Comment:
I should be able to click here to toggle the checkbox. You cheated by separating the checkbox from the text that goes with it. I assume it’s to line things up the way you like. Try another approach, or don’t require the text to line up?
![jcheckbox](https://user-images.githubusercontent.com/55099/29252683-6b99da78-8020-11e7-95a4-fbe1ac1857ff.png)